src/main/datatypes/MQ2GroupType.cpp

```diff
@@ -455,6 +750,64 @@ bool MQ2GroupType::GetMember(MQVarPtr VarPtr, const char* Member, char* Index, M
 			}
 		}
 		return false;
+		
+	case GroupMembers::Furthest:
+		Dest.DWord = 0;
+		Dest.Type = pFloatType;
+
+		for(auto& member : *pLocalPC->Group)
+		{
+			if(member && member->pSpawn)
+			{
+				auto distance = Distance3DToSpawn(pLocalPC->pSpawn, member->pSpawn);
```

Comment:
Since this is 3D, I'd probably call this Furthest3D or something (and the others). Normally we only calculate 2D distance for things like this.